### PR TITLE
Improved unit tests using mutation analysis

### PR DESCRIPTION
**Pull request by niels89**
Improved existing test suite and added tests to kill mutants resulting from mutation testing.

### CODE REVIEW

File: tests/de/gurkenlabs/litiengine/environment/tilemap/xml/BooleanIntegerAdapterTests.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package de.gurkenlabs.litiengine.environment.tilemap.xml;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
```
Comment (Gamebuster19901) [inline tests/de/gurkenlabs/litiengine/environment/tilemap/xml/BooleanIntegerAdapterTests.java:8]: Please remove the unused import `org.junit.jupiter.api.Assertions.assertFalse;`

**Review by Gamebuster19901 — CHANGES_REQUESTED** [review]

File: tests/de/gurkenlabs/litiengine/util/io/AlphanumComparatorTests.java [hunk 2]
```diff
@@ -1,6 +1,9 @@
 package de.gurkenlabs.litiengine.util.io;
 
+import de.gurkenlabs.litiengine.util.AlphanumComparator;
```
Comment (Gamebuster19901) [inline tests/de/gurkenlabs/litiengine/util/io/AlphanumComparatorTests.java:3]: Duplicate import `import de.gurkenlabs.litiengine.util.AlphanumComparator;` ????

**Review by Gamebuster19901 — CHANGES_REQUESTED** [review]

File: tests/de/gurkenlabs/litiengine/util/io/AlphanumComparatorTests.java (hunk 2 above)
Comment (jluech) [reply tests/de/gurkenlabs/litiengine/util/io/AlphanumComparatorTests.java:3]: probably a result of merge/rebase... damn "magic wand" :crossed_swords:

Comment (niels89) [reply tests/de/gurkenlabs/litiengine/util/io/AlphanumComparatorTests.java:3]: Removed the duplicated import

File: tests/de/gurkenlabs/litiengine/environment/tilemap/xml/BooleanIntegerAdapterTests.java (hunk 1 above)
Comment (jluech) [reply tests/de/gurkenlabs/litiengine/environment/tilemap/xml/BooleanIntegerAdapterTests.java]: removed 👍

**Review by nightm4re94 — APPROVED** [review]
LGTM - Thank you!

**Pull request merged**